Makefile

```diff
@@ -575,10 +575,17 @@ package:  ## Build rpm packages
 	rm -rf build/* *.src.rpm ~/rpmbuild/RPMS/*
 	./contrib/build_rpm.sh
 
+build/podman.repo: package
+
 # Remember that rpms install exec to /usr/bin/podman while a `make install`
 # installs them to /usr/local/bin/podman which is likely before. Always use
 # a full path to test installed podman or you risk to call another executable.
-package-install: package  ## Install rpm packages
-	sudo ${PKG_MANAGER} -y install ${HOME}/rpmbuild/RPMS/*/*.rpm
+package-install: package build/podman.repo  ## Install rpm packages
+	#sudo ${PKG_MANAGER} -y install ${HOME}/rpmbuild/RPMS/*/*.rpm
+	sudo cp -f build/podman.repo /etc/yum.repos.d/podman.repo
+	sudo ${PKG_MANAGER} -y --enablerepo podman install podman podman-remote
```

Comment:
This approach (repo disabled by default) is much better -- but the logical extension is to remove the `cp` entirely and use `--repofrompath` instead; this way the system isn't left in an unmaintainable state. Could you look into using that, esp. seeing if it works with yum?